### PR TITLE
doc: Add makechrootpkg man page

### PR DESCRIPTION
**Pull request by yoursweetie**
Add makechrootpkg man page.

### CODE REVIEW

**Comment by anthraxx** [comment]
was the close an accident?

**Comment by yoursweetie** [comment]
> was the close an accident?

Sorry, I found that more than a file is going to be added.

**Comment by anthraxx** [comment]
please just force push your feature branch instead of re-creating pull requests :cat:

**Comment by yoursweetie** [comment]
> please just force push your feature branch instead of re-creating pull requests

Sorry to disturb, I added other files which should be added & pushed them to my `add-doc-makechrootpkg` branch. Should I reopen this PR and do something else?

**Review by felixonmars — APPROVED** [review]

File: doc/makechrootpkg.1.asciidoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+makechrootpkg - Build a PKGBUILD in a given chroot environment
+
+Synopsis
+--------
+makechrootpkg [OPTIONS] -r <chrootdir> [--] [makepkg args]
+
+Description
+-----------
+
+Run this script in a PKGBUILD dir to build a package inside a
```
Comment (jelly) [inline doc/makechrootpkg.1.asciidoc:15]:
```suggestion
Run this script in a directory containing a PKGBUILD to build a package inside a
```

File: doc/makechrootpkg.1.asciidoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+*-I* <pkg>::
+	Install a package into the working copy of the chroot
+
+*-l* <copy>::
+	The directory to use as the working copy of the chroot
+	Useful for maintaining multiple copies
+	Default: $USER
+
+*-n*::
+	Run namcap on the package
```
Comment (jelly) [inline doc/makechrootpkg.1.asciidoc:65]:
```suggestion
	Run namcap on the build package
```

File: doc/makechrootpkg.1.asciidoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+*-C*::
+	Run checkpkg on the package
```
Comment (jelly) [inline doc/makechrootpkg.1.asciidoc:68]:
```suggestion
	Run checkpkg on the build package
```

**Review by jelly — CHANGES_REQUESTED** [review]
Apart from some small improvements, it looks good to me!

**Comment by jelly** [comment]
Thanks, merged via 80e8c1fc702e6a0e328bca52d3acf0710c7cc7ff

**Pull request closed without merging**